utils/utils.jsx

```diff
@@ -208,6 +218,15 @@ export function ding() {
     }
 }
 
+export function tryNotificationSound(name) {
```

Comment:
This could be reusable (if invoked by `ding()`) and perhaps simplified a bit:

```js
export function playNotificationSound(name) {
    const audio = new Audio(notificationSounds.get(name) ?? notificationSounds.get('Bing'));
    audio.play();
}
```

Reply:
Ok with this